services/QuillLMS/app/helpers/navigation_helper.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 module NavigationHelper
+  PAID = 'paid'
+  TRIAL = 'trial'
+  LOCKED = 'locked'
+  NONE = 'none'
+
   def home_page_should_be_active?
```

Comment:
I would remove the "should_be" language in all three of these methods since that language tends to be reserved for spec files:

home_page_should_be_active? -> home_page_active?
premium_page_should_be_active? -> premium_page_active?
quill_academy_should_be_active? -> quill_academy_active?